modules/exploits/windows/fileformat/zahir_enterprise_plus_csv.rb

```diff
@@ -0,0 +1,73 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
```

Comment:
Change `Msf::Exploit::Remote` to `Msf::Exploit`

Reply:
Ah this one, msftidy also complained about this problem. thanks.